scrapper/controllers/__init__.py

```diff
@@ -4,21 +4,17 @@
 from flask import render_template, request, redirect, Blueprint, abort
 from flask.ext.api import status
 
+from scrapper.controllers.twitter import twitter_bp
 from scrapper.controllers.miner import mine_user
 
 
-twitter = Blueprint('twitter', __name__, url_prefix = '/twitter')
-
-@twitter.route('/', methods=['GET', 'POST'])
+@twitter_bp.route('/', methods=['GET', 'POST'])
 def index():
     abort(400)
 
-@twitter.route('/<user_name>')
+@twitter_bp.route('/<user_name>')
 def show_user(user_name=None):
     refresh = request.args.get('refresh', '')
-    user = miner.mine_user(user_name, refresh)
-    user = json.dumps(user)
+    user = miner.mine_user.delay(user_name, refresh)
```

Comment:
We just want to spawn a task if the user does not exist in our database. If this is the case, we don't return a 200 Ok, but rather we just want to state that the request has been accepted and is being processed.

If the user already exists in our database, we don't spawn a new task, we simply return what we've got.